app/services/index.ts

```diff
@@ -23,6 +23,7 @@ export * from "./vm-size.service";
 export * from "./adal";
 export * from "./batch-client.service";
 export * from "./storage-client.service";
+export * from "./predefined-formula.service";
 
```

Comment:
yeah `tslint` doesn't check for the alpha ordering in`index.ts` files, not sure why